.github/workflows/deploy_functions.yml

```diff
@@ -1,6 +1,11 @@
-name: Deploy Functions Development
+name: Deploy Functions
```

Comment:
Couldn't we create a new workflow that gets together both "Deploy Functions" and "Deploy Terraform"? I think this way. We could simplify the workflow on the repository's side. It would be something like that:

```
name: Deploy Development
on:
  push:
    branches: [develop]

jobs:
  deploy-lambda-function:
    name: Deploy Lambda Function
    uses: groundbreaker/workflows/.github/workflows/deploy_lambda_function.yml@main
    with:
      environment: development
    secrets:
      GIT_CREDENTIALS: ${{ secrets.GIT_CREDENTIALS }}
      NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
      AWS_OPS_BUCKET: ${{ secrets.AWS_OPS_BUCKET }}
      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
      PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
```

Does it make sense?

Reply:
Can't you achieve that by using a [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action)?

Reply:
Oh yeah, maybe I can.
I will check that.